DSpace/DSpace
# Submission forms migration script

## References
* Fixes https://github.com/DSpace/DSpace/issues/2818

## Description
This PR introduces a script that will parse `input-forms.xml` and `item-submission.xml` from DSpace <7 versions into the `submission-forms.xml` and `item-submission.xml` standard for DSpace 7.
This script can be called by using `submission-forms-migrate -s {path-to-config-dir}/item-submission.xml -f {path-to-config-dir}/input-forms.xml`
with the `path-to-config-dir` leading to the config dir for the DSpace <7 instance that you're looking to migrate.
The migrated xml files will be present in the DSpace's install dir in the config folder

## Instructions for Reviewers

List of changes in this PR:
* introduced a script to automatically migrate the old submission xml files to the new ones

How to test this PR:
* Setup this branch
* Call the script using {dspace.dir}/dspace submission-forms-migrate -s {path-to-config-dir}/item-submission.xml -f {path-to-config-dir}/input-forms.xml
* Verify that the new submission xml files are present in the {dspace.dir}/config directory

## Checklist
